geometries/brian/M82pck.tpc

```diff
@@ -7,14 +7,14 @@ NH_TARGET_BODIES         += ( 9999000 )
 BODY9999000_POLE_RA        = (    0.    0.    0. )
 BODY9999000_POLE_DEC       = (   90.    0.    0. )
 BODY9999000_PM             = (  270.    0.    0. )
-BODY9999000_RADII          = (  946073047258080.  946073047258080.  946073047258080. )
```

Comment:
This was a mistake:  that value, 94...., repeated three times is 100ly in km.  and the _XYZ vector below is a unit vector, which makes no sense for an object relative the the solar system barycenter (SITE..._CENTER = 0)